credsweeper/__main__.py

```diff
@@ -168,7 +176,7 @@ def main() -> None:
         logging.info(f"Run analyzer on path: {args.path}")
         content_provider: FilesProvider = TextProvider(args.path, skip_ignored=args.skip_ignored)
         scan(args, content_provider, args.json_filename)
-    if args.diff_path:
+    elif args.diff_path:
```

Comment:
This change should be in the separate PR, I suppose

Reply:
Just to organize code. Arguments must be resolved in parcer
```
    group = parser.add_mutually_exclusive_group(required=True)
    group.add_argument("--path", nargs="+", help="file or directory to scan", dest="path", metavar="PATH")
    group.add_argument("--diff_path", nargs="+", help="git diff file to scan", dest="diff_path", metavar="PATH")
```